pysages/backends/openmm.py

```diff
@@ -176,17 +177,27 @@ def check_integrator(context):
         raise ValueError("Variable step size integrators are not supported")
 
 
-def bind(context, sampling_method, force = Force(), **kwargs):
+def bind(context, sampling_method, force=Force(), **kwargs):
+    """
+    Bind OpenMM as a backend to PySages.
+
+    context -- OpenMM context
+    sampling_method -- pysages sampling method
+    callback -- callback method with call signature `callback(snapshot, state)`
+      called after PySages updated. Example: logging of CVs.
+    """
     check_integrator(context)
-    #
+
     force.add_to(context)
+
     wrapped_context = ContextWrapper(context, force)
     helpers, bias = build_helpers(wrapped_context.view)
     snapshot = take_snapshot(wrapped_context)
     method_bundle = sampling_method(snapshot, helpers)
-    sync_and_bias = partial(bias, sync_backend = wrapped_context.synchronize)
-    #
-    sampler = Sampler(method_bundle, sync_and_bias)
+    sync_and_bias = partial(bias, sync_backend=wrapped_context.synchronize)
+    pysages_callback = kwargs.get("callback", None)
```

Comment:
As above, this won't be needed if we make `callback` an explicit argument to all `bind` methods.

Reply:
fixed